pkg/utils/utils.go

```diff
@@ -918,12 +917,16 @@ func GetBackupdriverClient(config *rest.Config) (v1alpha1.BackupdriverV1alpha1In
 }
 
 // Provide a utility function in guest cluster to clean up corresponding supervisor cluster snapshot CR
-func DeleteSvcSnapshot(gcSnapshot *backupdriverapi.Snapshot, svcConfig *rest.Config, svcNamespace string, logger logrus.FieldLogger) error {
-	svcSnapshotName := gcSnapshot.Status.SvcSnapshotName
+func DeleteSvcSnapshot(svcSnapshotName string, guestConfig *rest.Config, logger logrus.FieldLogger) error {
 	if svcSnapshotName == "" {
-		logger.Errorf("No corresponding supervisor cluster snapshot CR name for guest cluster snapshot %s", gcSnapshot.Name)
+		logger.Error("No corresponding supervisor cluster snapshot CR name for guest cluster snapshot")
```

Comment:
I think it's better to pass in namespace/name of gcSnapshot and log it for debugging purpose.

Reply:
Should we just print "Could not find snapshot CR of name %s", svcSnapshotName

Reply:
I think it's better to print out both namespace/name of gcSnapshot and svcSnapshot.

Reply:
Change to log format "namespace/name".